templates/sfn/ingest_sfn_definition.json.tpl

```diff
@@ -0,0 +1,153 @@
+{
+  "Comment": "${step_function_name}: A State machine for Judgment ingests",
+  "StartAt": "Map metadata",
+  "States": {
+    "Map metadata": {
+      "Type": "Task",
+      "Resource": "arn:aws:states:::lambda:invoke",
+      "OutputPath": "$.Payload",
+      "Parameters": {
+        "Payload.$": "$",
+        "FunctionName": "arn:aws:lambda:eu-west-2:${account_id}:function:${ingest_mapper_lambda_name}"
+      },
```

Comment:
For readability it may be easier to invoke the function by putting the function ARN in the `Resource` field. We can then lose the `OutputPath` and `Parameters` fields. Thoughts @MancunianSam?